db/seeds.rb

```diff
@@ -1,9 +1,42 @@
+  bio: "Hello, my name is Jang Na-Young and I'm 25 years old")
+            
+            
+# CREATE POSTS
+post = Post.create(author: first_user, title: "Why do I have to do this?", text: "Im really tired right now, but nobody cares so I have to keep going")
+post2 = Post.create(author: first_user, title: "Have you ever thought you were great?", text: "You know, have you ever thought that? I was thinking maybe I am")
+post3 = Post.create(author: first_user, title: "Too many meetings!!!", text: "Dude we're literally having 40 mandatory meetings a week, what the...?")
+post4 = Post.create(author: first_user, title: "I think the new coworker needs more training", text: "Im not hating or anything, but I believe Na-Young could benefit from some training")
+post5 = Post.create(author: first_user, title: "My boss is one of the best bosses I've ever had", text: "I almost love him lmfao")
+
+# CREATE COMMENTS
+comment = Comment.create(author: second_user, post: post, text:"Thats because you work here lmao, I literally pay you for that, you should sleep a little bit better though")
+comment2 = Comment.create(author: second_user, post: post2, text:"It'd be better if you stopped daydraming and finished the project, its due tomorrow")
+comment3 = Comment.create(author: first_user, post: post2, text:"I already finished thatm dont worry")
+comment4 = Comment.create(author: second_user, post: post2, text:"Oh I see, come to the office after lunch lol, we have to discuss next week schedule")
+comment5 = Comment.create(author: second_user, post: post3, text:"Only the necessary, please feel free to suggest changes if you dont agree with the schedule")
+comment6 = Comment.create(author: first_user, post: post3, text:"I already submitted the changes I believe would benefit the company lol")
+comment7 = Comment.create(author: third_user, post: post4, text:"Well I know I am not the best, but I AM doing my best")
+comment8 = Comment.create(author: first_user, post: post4, text:"Im just teasing you because I know you love me lol")
+comment9 = Comment.create(author: third_user, post: post4, text:"Lol dude you know the only thing I love is coffee")
+
+#CREATE LIKES
+like= Like.create(author: second_user, post: post)
+like2= Like.create(author: third_user, post: post)
+like3= Like.create(author: third_user, post: post2)
+like4= Like.create(author: third_user, post: post3)
+like5= Like.create(author: third_user, post: post4)
+like6= Like.create(author: third_user, post: post5)
+like7= Like.create(author: second_user, post: post2)
+like8= Like.create(author: second_user, post: post3)
+like9= Like.create(author: first_user, post: post5)
+like10= Like.create(author: second_user, post: post5)
```

Comment:
- [ ] Nice to add samples on the seed file. But it is not creating any sample. You can fix it. OPTIONAL
![image](https://user-images.githubusercontent.com/38635158/192070265-42e2e7be-57c0-4409-bf3b-03abe9b32df0.png)

___
### EXAMPLE:
![image](https://user-images.githubusercontent.com/38635158/192070314-2121b864-f18c-4eed-b2c8-82bb11bc37b1.png)